slic3r-prusa3d: 1.39.2 -> 1.40.0-beta

Moredread opened the pull request:
###### Motivation for this change

We need to fix the library installation path, as PERL_VENDORLIB and
PERL_VENDORARCH are set incorrectly. This can be seen when running cmake.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Moredread commented:
As it is a alpha version we might want to wait for the stable version before merging. But I already created this PR for the PERL_VENDORLIB and PERL_VENDORARCH fix, so that this isn't lost.

Moredread commented:
@tweber notifying you :) Shouldn't that happen automatically?

xeji commented:
We prefer to merge stable releases, so let's wait for it (it's 1.40.0-beta now). Marking this as WIP until then.

Moredread committed:
slic3r-prusa3d: 1.39.2 -> 1.40.0-beta

We need to fix the library installation path, as PERL_VENDORLIB and
PERL_VENDORARCH are set incorrectly. This can be seen when running cmake.

Moredread commented:
Package was updated in https://github.com/NixOS/nixpkgs/commit/dcb8828a8e9be79d56a38684afd9be8fb73697b5

Thanks @thorstenweber83!
